django_auth_adfs/backend.py

```diff
@@ -197,6 +197,10 @@ def process_access_token(self, access_token, adfs_response=None):
         if not claims:
             raise PermissionDenied
 
+        # Store tokens in session if middleware is enabled
+        if request and adfs_response and hasattr(request, "token_storage"):
+            request.token_storage.store_tokens(request, access_token, adfs_response)
```

Comment:
Would it be better to pull this out into a separate method that's called after `process_access_token`? It would make it easier for someone to customize what they want to do with the access token at that moment.